grafana/k6-studio
# feat: Persistent theme settings

## Description



## How to Test
* Try changing the default setting
* Close the app
* Open it again - the setting should persist

## Checklist



## Screenshots (if appropriate):
<img width="1402" alt="image" src="https://github.com/user-attachments/assets/d6ac63dc-a51e-4133-a0a0-20436971f519">


## Related PR(s)/Issue(s)
